src/cloud/setup/first-time-setup-import-import.md

```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
 
 When importing data, you will need to drop and create a new database. If you have done any data you want to keep, [create a backup]({{ site.baseurl }}/cloud/project/project-webint-snap.html) of the database.
 
+{:.procedure}
 To drop and re-create the Cloud database:
 
 1. SSH to the Integration environment.
 
-    ```
-    magento-cloud ssh
-    ```
+   ```bash
+   magento-cloud ssh
+   ```
 
 1. Connect to the database.
 
-    ```
-    mysql -h <db-host> -P <db-port> -p -u <db-user> <db-name>
-    ```
+   ```bash
+   mysql -h <db-host> -P <db-port> -p -u <db-user> <db-name>
+   ```
 
 1. Drop the database. At the `MariaDB [main]>` prompt, enter:
 
-    ```
-    drop database main;
-    ```
+   ```bash
+   drop database main;
+   ```
 
 1. Re-create the database:
 
-    ```
-    create database main;
-    ```
+   ```bash
+   create database main;
+   ```
 
 1. At the `MariaDB [main]>` prompt, enter `exit`.
 
 1. At the shell command prompt, enter the following command to re-create the database.
 
-    ```
-    zcat var/db.sql.tgz | sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' | mysql -h <db-host> -P <db-port> -p -u <db-user> <db-name>
-    ```
+   ```bash
+   zcat var/db.sql.tgz | sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' | mysql -h <db-host> -P <db-port> -p -u   <db-user> <db-name>
+   ```
 
-    For example,
+   For example,
 
-    ```
-    zcat var/db.sql.tgz | sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' | mysql -h database.internal -p -u user main
-    ```
+   ```bash
+   zcat var/db.sql.tgz | sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' | mysql -h database.internal -p -u user main
+   ```
 
 ### Update base URLs {#baseurl}
 
 Before you can access Magento from your local Cloud development system, you must change the Base URLs in the Magento database. Base URLs are stored in the `core_config_data` table.
 
 The following example shows how to change _only_ the insecure URL but you can use the same procedure to change secure URLs as well.
 
+{:.procedure}
 To update the unsecure base URL:
 
 1. If you haven't already done so, SSH to the Cloud integration server.
```

Comment:
There are a few of these intro sentences in this PR. Probably should check them for consistency.

```suggestion
1. If you have not already done so, use SSH to connect to the Cloud integration server.
```